### PR TITLE
About: no link if git revision unknown

### LINKED ISSUE #1686: About page should not show a link if git revision is "unknown"
### Steps to reproduce
Download .zip from GitHub (may not be a git clone).

### PR DESCRIPTION
**Pull request by plata**
fixes #1686

### CODE REVIEW

File: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/AboutPanel.java [hunk 1]
```diff
@@ -66,36 +54,39 @@ private void populate() {
...
-                opener.open(uri);
-            } catch (URISyntaxException e) {
-                e.printStackTrace();
-            }
-        });
-
-        this.buildTimestampDescription = new TextWithStyle(tr("Build Timestamp:"), "captionTitle");
-        this.buildTimestampLabel = new Label(buildInformation.getApplicationBuildTimestamp());
-
+        final Text nameDescription = new TextWithStyle(tr("Name:"), "captionTitle");
```
Comment (madoar) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/AboutPanel.java]: Maybe you should move the code to a new method:
```java
private GridPane createGridPane() {
   ...
}
```

**Comment by madoar** [comment]
As an alternative:
Do you want to try to reimplement the `AboutPanel` using MVC?
I know the documentation for MVC doesn't exist yet, but there should be enough examples.
Basically you would need to implement two classes:
- `public class AboutPanel extends ControlBase<AboutPanel, AboutPanelSkin>` and
- `public class AboutPanelSkin extends SkinBase<AboutPanel, AboutPanelSkin>`

File: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/AboutPanel.java [hunk 2]
```diff
@@ -20,106 +20,88 @@
...
-        this.aboutGrid.add(versionLabel, 1, 1);
+    private void populate(ApplicationBuildInformation buildInformation, Opener opener) {
+        Text title = new TextWithStyle(tr("About"), "title");
+
+        GridPane aboutGrid = new GridPane();
+        aboutGrid.getStyleClass().add("grid");
+        aboutGrid.setHgap(20);
+        aboutGrid.setVgap(10);
+
+        final Text nameDescription = new TextWithStyle(tr("Name:"), "captionTitle");
```
[the remark this answers is not in the document]
Comment (plata) [reply phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/AboutPanel.java]: Now I get:
```
WARNUNG: CSS Error parsing '*{captionTitle}: Expected COLON at [1,14]
```

[the remark this answers is not in the document]
Comment (madoar) [reply phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/AboutPanel.java]: `title.getStyleClass().addAll("captionTitle")`

**Pull request merged**